.github/workflows/lint.yml

```diff
@@ -0,0 +1,32 @@
+name: goreleaser
+
+on:
+  push:
+    branches: ['main', 'cpr-0']
```

Comment:
why limit to any branches at all? if you do
```
on:
  push
``` or w/e the exact syntax is should work too I think.

Reply:
Was used for testing, will remove.